services/brig/src/Brig/API/Public.hs

```diff
@@ -621,7 +621,7 @@ newNonce :: UserId -> (Handler r) Nonce
 newNonce _ = do
   ttl <- setNonceTtlSecs <$> view settings
   nonce <- randomNonce
-  lift $ wrapClient $ Nonce.addNonce ttl nonce
+  lift $ wrapClient $ Nonce.insertNonce ttl nonce
```

Comment:
interesting, `delete` is not called anywhere.  have you tested it manually?  (not that important, it'll happen soon, might as well be in a separate PR.)